doorway-ui-components/src/page_components/listings/ZeroListingsItem.scss

```diff
@@ -0,0 +1,30 @@
+.zero-listings {
+    --margin: var(--bloom-s3);
+    --bottom-margin: var(--bloom-s6);
+    --border-color: var(--bloom-color-gray-450);
+    --padding-top: var(--bloom-s3);
+    --padding-bottom: var(--bloom-s6);
+  
+    margin-top: var(--margin);
+    margin-left: var(--margin);
+    margin-right: var(--margin);
+    margin-bottom: var(--bottom-margin);
+    padding-top: var(--padding-top);
+    padding-bottom: var(--padding-bottom);
+    list-style-type: none;
+    border-bottom: 1px solid var(--border-color);
+  }
+
+.zero-listings-description {
+    --padding-top: var(--bloom-s3);
```

Comment:
(another nit) I think that the more common pattern I see are these variables stored at the top level class? with some namespacing so like "--description-padding-top" and "--item-padding-top" to avoid name colissions